docs/deployment/landing_page.rst

```diff
@@ -120,7 +120,7 @@ services intercept requests between a potential source and the SecureDrop
 
 Do Not Hyperlink .onion Addresses
 ---------------------------------
-Because a visitor to your *Landing Page* may not be using the Tor browser yet,
+Because a visitor to your *Landing Page* may not be using the Tor Browser yet,
```

Comment:
Nit: if we're capitalizing the product name, we should IMO not also use the definite article (`the`), i.e. I would suggest changing such instances to "may not be using Tor Browser yet", in the same way you would say "may not be using Firefox" instead of "may not be using the Firefox".

Reply:
I'll remove the articles in front of "Tor Browser" :+1: